test/OpenTelemetry.Tests.Stress.Metrics/OpenTelemetry.Tests.Stress.Metrics.csproj

```diff
@@ -6,6 +6,10 @@
     <TargetFrameworks>net6.0;netcoreapp3.1;net462</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry.Instrumentation.Runtime" Version="0.2.0-alpha.1" />
```

Comment:
might be movable to common.nonprod.props, but will first see if we can move the project to main repo, and use project reference.